src/types/NewsroomLanguageSettings.ts

```diff
@@ -14,4 +15,5 @@ export interface NewsroomLanguageSettings {
     default_cookie_statement: string;
     default_subscribe_disclaimer: string;
     notifications: Notification[];
+    seo_settings: SEOSettings;
```

Comment:
Note that this is a different API to get the language SEO settings (Newsroom Languages API), than it is used to update this settings (Newsrooms API).

Reason why I did this:
1. Single `Save` button on frontend side, so doesn't make sense to do two separate API requests to update the whole form (one to Newsrooms API second to Newsroom Languages API).
2. It would add to the newsroom payload if we would introduce a property that would contain settings per language.

@aspirisen let me know what you think of this and if this is ok to use on frontend?

Reply:
Am i right that when we need to update settings seo page i will use Newsroom client and call `update` method with data from the the settings page using `google_search_console_key` and `cultures_seo_settings` from UpdateRequest type.

When i load the page i need to use  NewsroomLanguages client and call `get` method ?

Also, what field i should use to save and load this flag?
<img width="720" alt="image" src="https://user-images.githubusercontent.com/3620639/207052330-9ba7c4c6-7f6d-4473-9d91-3703cad10fd4.png">

Reply:
@aspirisen that is correct, as you described. Would that be a problem that there is a different API to get this data? You think it's not a good practice, or have proposal to do it differently?

> Also, what field i should use to save and load this flag?

It's `is_indexable` and it already exists. We are using it in newsroom advanced settings, so we are basically just moving it as I understood.
<img width="1293" alt="Screenshot 2022-12-12 at 14 19 49" src="https://user-images.githubusercontent.com/15163962/207055069-845fadee-9f00-4993-9834-d45b80123801.png">

Reply:
> Would that be a problem that there is a different API to get this data?
i think that's ok

one problem is that i will need to make two requests to load the page, one for `is_indexable` and another one to load everything else, but i think it is not a problem